nengo_viz/static/viz_netgraph.js

```diff
@@ -298,22 +300,40 @@ VIZ.NetGraph.prototype.set_scale = function(scale) {
 
 
 VIZ.NetGraph.prototype.update_font_size = function(scale) {
-    // TODO: text resizing is disabled for now until we can figure out
-    // how best to have it work with large models
+    if (this.zoom_fonts) {
+        $('#main').css('font-size', 3 * this.scale * this.font_size/100 + 'em');
```

Comment:
Do we want to keep that factor of 3 or should we make the font-size match independent of the zoom mode on scale = 1.